base/server/certs/device.conf

```diff
@@ -0,0 +1,9 @@
+# OpenSSL certificate extension configuration for device certificates
+# Used for IoT devices, client authentication, and other non-user endpoints
+# Appropriate for EST fullcmc device enrollment and similar use cases
```

Comment:
The current files in [base/server/certs](https://github.com/dogtagpki/pki/tree/master/base/server/certs) define the extension for PKI system certs. Since they are generally not customizable they will only be installed in the shared folder (i.e. `/usr/share/pki/server/certs`) and not in the instance's config folder.

IIUC the `device.conf` seems to be some thing that a CA/EST admin would customize, so I'd suggest to move it into [base/ca/shared/conf](https://github.com/dogtagpki/pki/tree/master/base/ca/shared/conf) together with other files that will be installed in `<instance>/conf/ca`.

Reply:
IIUC these files are used to generate CSR from "clients" so I do not think they should go inside the instance where the configuration files are for the server operations. They should stay in shared folder but eventually in a different (sub-)folder.

Reply:
I think the `device.conf` could stay here and be referred  in the documentation. However,, if you agree to remove and add the documentation details to create from scratch  it is OK with me.

Reply:
Thanks for the clarification. Will the person using the `device.conf` likely have access to server files? If not, then the file probably should not be stored under `/usr/share/pki/server` since they are part of `dogtag-pki-server` package or the subsystem packages. How about these locations instead?
* https://github.com/dogtagpki/pki/tree/master/base/common/examples
* https://github.com/dogtagpki/pki/tree/master/base/tools/examples